pyppeteer/execution_context.py

```diff
@@ -29,209 +28,107 @@
 class ExecutionContext(object):
     """Execution Context class."""
 
-    def __init__(self, client: CDPSession, contextPayload: Dict,
-                 objectHandleFactory: Any, frame: 'Frame' = None) -> None:
+    def __init__(
+            self,
+            client: CDPSession,
+            contextPayload: Dict,
+            world: 'DOMWorld',
```

Comment:
I think we should come to a bit of a standard on whether to use strings in typehints. Right now I'm leaning towards just quoting the type as it doesn't really reduce development friciton - things like 'Navigate to definition' in PyCharm still work as expected.

Reply:
AFAIK strings solve circular imports which I assume was intial intention for project though if we can confirm that circular imports aren't happening I'd be more than happy to migrate to class imports as it just looks nicer and probably works better with other editors.